src/App.js

```diff
@@ -125,9 +125,13 @@ const App = ({ locale, location, window, dependencies: { appSessionRef, sessionM
                 component={SubscribersLegacyUrlRedirect}
               />
               <PrivateRoute path="/new-features" exact component={NewFeatures} />
-              <PrivateRoute path={['/plan-selection']} exact component={ChangePlan} />
               <PrivateRoute
-                path={'/plan-selection/:pathType/:planType?'}
+                path={['/plan-selection-deprecated']}
+                exact
+                component={ChangePlanDeprecated}
+              />
+              <PrivateRoute
+                path={'/plan-selection-deprecated/:pathType/:planType?'}
                 exact
                 component={PlanCalculator}
```

Comment:
Is PlanCalculator not deprecated?

Reply:
![image](https://user-images.githubusercontent.com/84402180/136721300-4a312b88-b13c-40d0-943f-5ed64fdbce0b.png)
![image](https://user-images.githubusercontent.com/84402180/136721382-4d9d9918-7930-496b-b40c-25dcd9374464.png)

Reply:
![image](https://user-images.githubusercontent.com/84402180/136721534-8788b843-1208-461b-b93e-f26088baa15c.png)
![image](https://user-images.githubusercontent.com/84402180/136721558-f92bd95c-0758-4b03-b71e-094c8e963501.png)